l10n_ec_ote/data/res.bank.csv

```diff
@@ -0,0 +1,93 @@
+id,country:id,name,bic,active
```

Comment:
You can update your file and only keep records that don't currently exist in the odoo database, see file https://github.com/odoo/odoo/blob/15.0/addons/l10n_ec/data/res.bank.csv